mix.exs

```diff
@@ -29,11 +29,11 @@ defmodule JWKSURIUpdater.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:dialyxir, "~> 1.0.0-rc.4", only: [:dev], runtime: false},
-      {:ex_doc, "~> 0.19", only: :dev, runtime: false},
+      {:dialyxir, ">= 0.0.0", only: [:dev], runtime: false},
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
       {:hackney, "~> 1.0", only: :dev},
       {:jose_utils, "~> 0.1"},
-      {:poison, "~> 4.0"},
+      {:poison, "~> 3.0 or ~> 4.0 or ~> 5.0", optional: true},
```

Comment:
We can actually remove the poison dependency. I think this was the default Tesla decoder at some point, but they switched to Jason, which is a dependency of `jose_utils`. So no need for `poison` anymore.

Reply:
Great point :sun_with_face: I've removed `:poison` and updated the PR.